ci-operator/jobs/openshift/release/openshift-release-release-4.4-periodics.yaml

```diff
@@ -558,10 +580,30 @@ periodics:
       - name: CLUSTER_TYPE
         value: aws
       - name: CONFIG_SPEC
-        valueFrom:
-          configMapKeyRef:
-            key: openshift-origin-release-4.4.yaml
-            name: ci-operator-4.4-configs
+        value: |
```

Comment:
why is this needed?

Reply:
oh is it because it would otherwise run _all_ the tests, whereas we just to run the e2e test?

Reply:
You don't need to run all tests, that is what `--target TEST_NAME` does.

Reply:
A custom `CONFIG_SPEC` section is required to import machine-os-content from `fcos` (see `base_images` there) and `promotion` section (added `--promote` option to ci-operator)